src/components/AITypingIndicatorView/hooks/useAIState.ts

```diff
@@ -0,0 +1,60 @@
+import { useEffect, useState } from 'react';
+
+import { Channel, Event } from 'stream-chat';
+
+import type { DefaultStreamChatGenerics } from '../../../types/types';
+
+export type AIStateType =
```

Comment:
Could this type be take from the client instead of duplicating it in the component SDK?

Reply:
Yep, was not available in the client up until this morning